app/jobs/bulk_update/placements/import_row_job.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module BulkUpdate
+  module Placements
+    class ImportRowJob < ApplicationJob
+      queue_as :bulk_update
+
+      def perform(placement_row)
+        ImportRow.call(placement_row)
+      rescue StandardError => e
```

Comment:
Could we rescue the specific error?

Reply:
for now I'm not sure what may crop up - could get more specific once we've had it running for real?

Reply:
Sounds good